### ISSUE TITLE
Minimum transfer amount

### ISSUE THREAD

**Issue by mejustandrew** [steemit/steem#2467]
Right now the code does not limit the amount of Steem/SBDs to be sent from an account to another, making it easy to spam with memos and phishing links, and also with so called services that are meant to scam users of their money.

A small adjustment in the [steem_evaluator](https://github.com/steemit/steem/blob/master/libraries/chain/steem_evaluator.cpp) could adjust the minimum amount of money sent to 0.005 Steem/SBD.

![image](https://user-images.githubusercontent.com/36562241/40206638-a2d7fd00-5a39-11e8-9b48-546bbdc830bf.png)

This will discourage people phishing and also accounts that are following new users only to get followers back in order to trick people for payed resteems, like [hottopic](https://steemit.com/@hottopic) and will reduce this way a bit the load on the blockchain, leaving more bandwidth for the average user.

There are many upsides in this limit of transfer, even though there are downsides too like some fair services that are sending money in order to notice users as [steemcleaners](https://steemit.com/@steemcleaners) does right now, will need to adjust the sum of money they are sending to the users.

**Comment by shredz7** [comment]
I agree with this idea. But what if the minimum send value is 0.001 Steem/SBD but the minimum send value *with a memo* is 0.005? This would allow for micropayments while also making sure that it is harder to just send random messages to people.

**Comment by Gandalf-the-Grey** [comment]
It wouldn't solve the issue, it will just move it / postpone it for a while.
I think a better way for solving spam issue would be to re-design of bandwidth algorithm.
Think about STEEM being worth 10USD, 50USD or 100USD.
Actually I would rather see the opposite - precision increased to 0.0001 or 0.00001, in such cases we still need to restrict spamming without blocking micro-payments.
Some of related issues can be solved on UI/UX level (in condenser, etc.) such as
https://github.com/steemit/condenser/issues/2226 etc.

**Comment by mejustandrew** [comment]
I can see your concern, GtG, but even since I opened the issue, I thought about the micro-payments problem. Even for $100/Steem, allowing for a minimum transfer amount of 0.001 to be sent, will still allow users to send 10 cents to each other, which I guess it is enough for micro-payments. Even a coffee or a bread in not so developed countries is more than that. And Steem is still a blockchain that does not have fees. A lot of other blockchains do not offer these capabilities.

This combined with the proposal made by shredz7 will most likely solve the issue we have right now. Lately there are a lot of accounts made only to create resteem bots, and this will become a huge problem with the bandwidth much faster than Steem will face the problem of micro-payments.
